### ISSUE TITLE
Faster Descend

### ISSUE THREAD

**Issue by HascEasyZ** [444A49/minifindings#12]
I cant seem to get the descend speed faster using the instructions.
Hover height etc works fine but the drone simply wont go down faster then 12km/h

Anything i am missing maybe?

**Comment by mihaigabriel620** [comment]
yes you must change one more parameter to arrive to a higher descend speed ( recomend using dronehacks software )

( vert_vel_down_adding_max )

if you set it to -10 also it can reach 35km/h vertical speed descend

i hope it is simple to understand😎

**Comment by mihaigabriel620** [comment]
> I cant seem to get the descend speed faster using the instructions. Hover height etc works fine but the drone simply wont go down faster then 12km/h
> 
> Anything i am missing maybe?

yes you must change one more parameter to arrive to a higher descend speed ( recomend using dronehacks software )

( vert_vel_down_adding_max )

if you set it to -10 also it can reach 35km/h vertical speed descend

i hope it is simple to understand😎
